inginious/frontend/pages/course_admin/aggregation_list.py

```diff
@@ -127,6 +128,14 @@ def page(self, course, msg="", error=False):
                 other_aggregations.append(aggregation)
 
         if "csv" in web.input():
+            for value in data:
+                value["task_name"] = self._get_task_name(course, value["_id"])
             return make_csv(data)
 
-        return self.template_helper.get_renderer().course_admin.aggregation_list(course, [my_aggregations, other_aggregations], msg, error)
+        return self.template_helper.get_renderer().course_admin.aggregation_list(course,
+                                                                                 [my_aggregations, other_aggregations],
+                                                                                 msg, error)
+
+    def _get_task_name(self, course, task_id):
+        """ Return the name of a task if it isn't empty, else keeps the id """
+        return course.get_task(task_id).get_name(self.user_manager.session_language())
```

Comment:
What if the task does not have a name? In that case we should show the task id.

Reply:
The CSV already has the id, so instead of replacing it, I add a new column